packages/ui/src/helpers/authenticator/utils.ts

```diff
@@ -4,12 +4,22 @@
  */
 
 import { Hub } from 'aws-amplify';
+import { appendToCognitoUserAgent } from '@aws-amplify/auth';
 import { waitFor } from 'xstate/lib/waitFor';
 
 import { AuthInterpreter, AuthMachineHubHandler } from '../../types';
 import { ALLOWED_SPECIAL_CHARACTERS } from './constants';
 import { getActorState } from './actor';
 
+type ConfigureOptions = { packageName: string; version: string };
+export const configureComponent = ({
+  packageName,
+  version,
+}: ConfigureOptions) => {
+  // "@aws-amplify/ui-react" + "/" + "3.5.10"
```

Comment:
Is this comment necessary?

Reply:
Not totally, but it is intentional to let future versions of ourselves remember what this is for